src/store/holochain.js

```diff
@@ -36,6 +35,7 @@ const initializeClientHolo = async (commit, dispatch, state) => {
   let holoClient
 
   if (!state.holoClient) {
+    console.log('CHAPERONE_SERVER_URL : ', process.env.VUE_APP_CHAPERONE_SERVER_URL)
```

Comment:
I actually moved this here bc it isn't a relevant log unless we're in the holo env.  It could be confusing to see this log whenever we're in the holochain context.  Thoughts?

Reply:
Not a big deal, but my preference would be to put it at the top but just check that the env var is defined.
```
if (process.env.VUE_APP_CHAPERONE_SERVER_URL !== undefined) {
  console.log('CHAPERONE_SERVER_URL', process.env.VUE_APP_CHAPERONE_SERVER_URL)
}
```

The reasoning is just to keep the constant logs in one place and to keep the logic as uncluttered as possible.

Reply:
Ok I could go either way on this one.  I'll go ahead and move it since you have that preference.